components/omega/configs/Default.yml

```diff
@@ -25,3 +25,60 @@ Omega:
     ViscDel2: 1.0e3
     VelHyperDiffTendencyEnable: true
     ViscDel4: 1.2e11
+  IOStreams:
+    InitialState:
+      UsePointerFile: false
+      Filename: OmegaMesh.nc
+      Mode: read
+      Precision: double
+      Freq: 1
+      FreqUnits: OnStartup
+      UseStartEnd: false
+      Contents:
+        - Restart
+    RestartWrite:
+      UsePointerFile: true
+      PointerFilename: ocn.pointer
+      Filename: ocn.restart.$Y-$M-$D_$h.$m.$s
+      Mode: write
+      IfExists: replace
+      Precision: double
+      Freq: 6
+      FreqUnits: months
+      UseStartEnd: false
+      Contents:
+        - Restart
+    RestartReadTest:
+      UsePointerFile: true
+      PointerFilename: ocn.pointer
+      Mode: read
+      Precision: double
+      Freq: 6
+      FreqUnits: months
+      UseStartEnd: false
+      Contents:
+        - Restart
```

Comment:
@xylar Yes, that is what I was struggling with here. We want InitialState to be for a startup from scratch. But once the model is going, we want to use Restarts with a pointer file to read the initial state. I put them both in Default to illustrate and test the two cases. The unit tests need to start using the InitialState, but I wanted to also test the restart and pointer file reads, so I had it in the config as a test.  Unfortunately, since I define streams only in the streams config, I didn't include a way to add a stream internally for a unit test.

Reply:
@xylar One way to have them both in the Default would be to have the RestartRead turn on UseStartEnd with a start date just slight after the initial time and the end date far in the future. Or I could add a StartAfter flag...

Reply:
> One way to have them both in the Default would be to have the RestartRead turn on UseStartEnd with a start date just slight after the initial time and the end date far in the future.

I kind of like this approach. It seems intuitive to me. And then maybe if someone _doesn't_ want to restart an existing run they set the read frequency to never.

Reply:
That also sounds reasonable to me.

Reply:
I think it would be a little cleaner in Polaris if we are explicit about when we want to read from the initial condition vs. a restart file.  I don't think that's too much to expect from a user as long as we document how it's done.

Reply:
Yes and I will change. I had it set this way so it would immediately read/test the restart that was just written, but for the Default yaml file, we only want read once at startup. Thanks.

Reply:
@xylar @cbegeman I just pushed a change to Default.yml with comments and stream changes that reflect the discussion above. Let me know if you want further changes

Reply:
This looks right to me. Thanks @philipwjones! And let me know if you or @xylar would like me to do any testing.

Reply:
Yes, I agree, I like the changes you made to default.yml. Looks good!